Tools/CISupport/ews-build/config.json

```diff
@@ -349,9 +349,9 @@
     {
       "type": "AnyBranchScheduler", "name": "pullrequest",
       "builderNames": [
-            "Bindings-Tests-EWS", "iOS-15-Build-EWS",
```

Comment:
GTK builder queue triggers layout-test and api tests queues as well. Those queues doesn't seem ready yet.